site/src/atomics/image.md

```diff
@@ -10,7 +10,7 @@ classPrefixes:
 
 # Image
 
-Image atomics are used to set the image behavior of an element.
+Image atomics are used to set the image behavior of an element. Image height and width ratios can be set with [aspect ratio atomics](./aspect-ratio.md).
```

Comment:
Ah sorry @jwmunn I originally meant that this warranted mentioning in the image component. I think its definitely fine to keep this line here, but it needs to be added to this page as well: https://design.learn.microsoft.com/components/image.html